op-node/flags/flags.go

```diff
@@ -243,6 +243,30 @@ var (
 		EnvVars: prefixEnvVars("L2_BACKUP_UNSAFE_SYNC_RPC_TRUST_RPC"),
 		Hidden:  true,
 	}
+	ConductorEnabledFlag = &cli.BoolFlag{
+		Name:    "conductor.enabled",
+		Usage:   "Enable the conductor service",
+		EnvVars: prefixEnvVars("CONDUCTOR_ENABLED"),
+		Value:   false,
+	}
+	ConductorAddrFlag = &cli.StringFlag{
+		Name:    "conductor.addr",
+		Usage:   "Conductor service rpc address",
+		EnvVars: prefixEnvVars("CONDUCTOR_ADDR"),
+		Value:   "localhost",
+	}
+	ConductorPortFlag = &cli.IntFlag{
+		Name:    "conductor.port",
+		Usage:   "Conductor service rpc port",
+		EnvVars: prefixEnvVars("CONDUCTOR_PORT"),
+		Value:   8547,
+	}
+	ConductorRpcTimeoutFlag = &cli.DurationFlag{
+		Name:    "conductor.rpc-timeout",
+		Usage:   "Conductor service rpc timeout",
+		EnvVars: prefixEnvVars("CONDUCTOR_RPC_TIMEOUT"),
+		Value:   time.Millisecond * 100,
```

Comment:
IMO this is a pretty low default value.

Reply:
Can we make it higher to like 1 second? having default value to be a little bit looser would make me feel better (to avoid random delays causing rpc failures)

Reply:
fair, bumped it to 1 second